JSDemos/Demos/List/ItemTemplate/description.md

```diff
@@ -1 +1,3 @@
-This demo shows how you can apply a&nbsp;custom template to&nbsp;items in&nbsp;the **List** UI component.
\ No newline at end of file
+The **List** supports item customization. You can use predefined customization settings. To do so, define [specific fields](/Documentation/ApiReference/UI_Components/dxList/Configuration/items/) in item data object.
+
+In cases when you need deeper customization, use the [itemTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemTemplate) property illustrated in this demo.
```

Comment:
Since in my version, custom templates are mentioned as a second way of customization:

```suggestion
In cases when you need deeper customization, implement a custom template and assign it to the [itemTemplate](/Documentation/ApiReference/UI_Components/dxList/Configuration/#itemTemplate) property, as illustrated in this demo.
```